package.json

```diff
@@ -522,13 +522,17 @@
         ]
     },
     "scripts": {
-        "vscode:prepublish": "webpack --mode production",
-        "webpack": "webpack --mode production",
+        "install:all": "npm install && cd webview-ui && npm install",
+        "dev:webview": "cd webview-ui && npm run dev",
+        "build:webview": "cd webview-ui && npm run build",
+        "vscode:prepublish": "npm run webpack",
+        "webpack": "npm run build:webview && webpack --mode production",
         "webpack-dev": "webpack --mode development --watch --info-verbosity verbose",
-        "test-compile": "tsc -p ./",
+        "test-compile": "npm run build:webview && tsc -p ./",
         "compile": "webpack --mode none",
         "watch": "tsc -watch -p ./",
-        "test": "mocha -V && npm run test-compile && node ./out/tests/runTests.js"
```

Comment:
I thought `mocha -V` was to check whether `mocha` is installed globally, and that it's not required because it's installed as a development dependency, so I removed it. But actually it worked fine for me before that change, even though I don't have it installed globally. I still _think_ calling `mocha -V` is not necessary, but I can add it back if it's there for a good reason I don't understand.